hypertrace-view-generator/src/main/resources/configs/view-gen-backend-entity/application.conf

```diff
@@ -18,10 +18,12 @@ flink.source {
   input.class = org.hypertrace.core.datamodel.StructuredTrace
   schema.registry {
     schema.registry.url = "http://localhost:8081"
+    schema.registry.url = ${?SCHEMA_REGISTRY_URL}
     specific.avro.reader = true
   }
   kafka {
     bootstrap.servers = "localhost:9092"
+    bootstrap.servers = ${?BOOTSTRAP_SERVERS}
```

Comment:
prefix env name with KAFKA_

Reply:
As discussed offline, make sense, will do this.

Reply:
Done as discussed by prefixing with the underlying component for consistency and context.